Mathlib/Topology/DenseEmbedding.lean

```diff
@@ -27,9 +27,7 @@ has to be `DenseInducing` (not necessarily injective).
 
 noncomputable section
 
-open Set Filter
-
-open Classical Topology Filter
+open Set Topology Filter
```

Comment:
These pairs of `open`s usually appear because `mathport` ported `open_locale` as `open` instead of `open scoped`.
```suggestion
open Set Filter
open scoped Topology
```